tests/unit/strategy/pool_heuristics_tests.cpp

```diff
@@ -119,3 +119,32 @@ TYPED_TEST(PoolHeuristicsTest, BlocksReleasable)
   ASSERT_EQ(a.second->getReleasableBlocks(), 1);
   ASSERT_EQ(a.second->getTotalBlocks(), 1);
 }
+
```

Comment:
I suggest that you leave a comment in here about the assumptions this test is making about the size of the blocks and what is being done to account for any alignment that the underlying allocator is doing.  I read through the code and can see how it would work, but leaving a few breadcrumb comments might help me when/if I look at it again a few years from now :-).

Reply:
For sure! For my test I also allowed the pool to coalesce itself and did not manually call the coalesce. function But to do this I had to make 3 blocks releasable not 2 although the heuristic I used was `blocks_releasable_hwm(2)`. I explain this discrepancy in my comment below.